test/fixtures/import.json

```diff
@@ -0,0 +1,30 @@
+{
+	"syntax": 3,
+	"package": null,
+	"imports": ["./result.proto", "./other_result.proto"],
+	"enums": [],
+	"extends": [],
+	"messages": [
+		{
+			"name": "SearchResponse",
+			"extensions": null,
+			"enums": [],
+			"extends": [],
+			"options": {},
+			"messages": [],
+			"fields": [
+				{
+					"name": "result",
+					"type": "Result",
+					"tag": 1,
+					"map": null,
+					"oneof": null,
+					"required": false,
+					"repeated": true,
+					"options": {}
+				}
+			]
+		}
+	],
+	"options":{}
+}
```

Comment:
can you indent this by 2 spaces instead of a tab to make it consistent with the other fixtures?

Reply:
Definitely can do that. I got lazy and didn't set an editor config when making my changes :)

Reply:
OK, think I've got it set up with two spaces rather than a tab.

Reply:
Perfect